src/scss/_themes.scss

```diff
@@ -63,7 +66,7 @@
 				padding: 0 10px;
 			};
 
-			&:nth-child(4) {
+			&:last-child {
```

Comment:
Or are you referring to a comment as to _why_ I made a change, not what that bit is doing?